docs/workflows/lambda.md

```diff
@@ -57,6 +57,10 @@ For new Node.js lambdas, update `<daac>-deploy/lambdas.yml` by adding a new entr
       handler: <dir>.<function>                     # eg:  sample-lambda.handler (assuming file has module.exports.handler = <someFunc>)
       timeout: <s>                                  # eg:  300
       source: 'node_modules/@cumulus/<dir>/dist/'   # eg:  '../cumulus/cumulus/tasks/sample-lambda/dist/index.js'
+      useMessageAdapter: true                       # necessary if this Lambda is included as part of a Cumulus workflow
```

Comment:
FWIW, these options are documented below: https://github.com/nasa/cumulus/pull/1137/files#diff-a2f26960b6993fff004993a1a0fc04bbL83

I can see the argument for having them documented directly in these code blocks though. Maybe we should we remove `useMessageAdapter` and `layers` from the "otehr options" block below and only include them in these example lambda configs to avoid duplication?

Reply:
Good point.  How about 552de1492?